src/services/fluence-aurora-connector/modules/fluence_aurora_connector/src/chain/mod.rs

```diff
@@ -1,6 +1,9 @@
 pub mod chain_data;
 pub mod chain_event;
+pub mod chain_info;
+pub mod data_tokens;
 pub mod deal_changed;
 pub mod deal_created;
```

Comment:
Do I understand correctly that there will be no event of this type and it can be removed?

Reply:
Yes, it can be removed.

Reply:
But I think it shouldn't be removed, for a while. Will remove it in the next PR.